values.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Use Palisade-services values.yaml mostly, but set some useful defaults here
+
+# Used by templates for naming resources, see assorted helpers.tpl
+fullnameOverride: palisade-examples
+
+global:
+  # uniqueNamespace: if true (and this is a new install) the generated deployment name will be used as the namespace name too
+  uniqueNamespace: false
```

Comment:
Am not sure on this mechanism for namespacing, can we just have the user supply a --namespace my-namespace when they do the helm update and it work out if you need a new namespace or not. And we can default to palisade-examples as the namespace if none is provided.

Reply:
updated